pentaho-platform-plugin-deployer/src/main/java/org/pentaho/osgi/platform/plugin/deployer/impl/PluginZipFileProcessor.java

```diff
@@ -328,6 +338,42 @@ public boolean accept( File f ) {
     }
   }
 
+  public void processManifest( ZipInputStream zipInputStream, ZipOutputStream zipOutputStream ) throws Exception {
+
+    File dir = Files.createTempDir();
+    PluginMetadata pluginMetadata = null;
+    try {
+      pluginMetadata = new PluginMetadataImpl( dir );
+    } catch ( ParserConfigurationException e ) {
+      throw new IOException( e );
+    }
+
+    Manifest manifest = null;
+
+    try {
+      Set<String> createdEntries = new HashSet<String>();
```

Comment:
This `Set` isn't used anywhere (apart from the `add` in line 360 that serves no propose).